Godeps/Godeps.json

```diff
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "github.com/letsencrypt/boulder",
 	"GoVersion": "go1.5",
-	"GodepVersion": "v67",
+	"GodepVersion": "v69",
```

Comment:
Yeah - this just happens to be the version of Godep I have installed. We have some code now to specifically ignore the saved Godep version, so we don't really worry about what it gets updated to.